core/lowering/lowering.h

```diff
@@ -6,12 +6,18 @@ namespace trtorch {
 namespace core {
 namespace lowering {
 
+struct LowerInfo {
+  // Internal flag to ensure torch.jit.Module does not get freezed in lowering.cpp. This is required for QAT models.
+  bool unfreeze_module;
+};
+
 void LowerBlock(torch::jit::Block* b);
-void LowerGraph(std::shared_ptr<torch::jit::Graph>& g);
+void LowerGraph(std::shared_ptr<torch::jit::Graph>& g, bool disable_cse /*=false*/);
 torch::jit::Module LowerModule(const torch::jit::script::Module& mod);
 std::pair<std::shared_ptr<torch::jit::Graph>, std::vector<torch::jit::IValue>> Lower(
     const torch::jit::script::Module& mod,
-    std::string method_name);
+    std::string method_name,
+    bool unfreeze_module /*=false*/);
```

Comment:
Is the default supposed to be in the forward declaration or in the implementation